code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -156,6 +156,29 @@
 			. = 1
 	..()
 
+/datum/reagent/medicine/clonexadone
+	name = "Clonexadone"
+	id = "clonexadone"
+	description = "A chemical that derives from Cryoxadone. It specializes in healing clone damage, but nothing else. Requires very cold temperatures to properly metabolize, and metabolizes quicker than cryoxadone."
+	color = "#0000C8"
+	taste_description = "muscle"
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/clonexadone/on_mob_life(mob/living/M)
+	switch(M.bodytemperature) // Low temperatures are required to take effect.
+		if(0 to 100) // At extreme temperatures (upgraded cryo) the effect is greatly increased.
```

Comment:
oh wait so you didn't actually re-add clonex as it was originally you just made it handle specifically clone stuff?

this is much better but also just feels really needless